### ISSUE TITLE
Sam Csik blog post - Aug 3

### ISSUE THREAD

**Issue by jules32** [Openscapes/how_we_work#266]

**Comment by jules32** [comment]
Hi @stefaniebutland, I gave Sam minor feedback in this issue, if you'd like to follow suit https://github.com/samanthacsik/samanthacsik.github.io/pull/27. 

Before cross-posting on the Openscapes blog, could you add a short italics at the top introducing Sam and connect to her leading the Quarto community call? Thank you!

**Comment by samanthacsik** [comment]
Just a note here on post updates!

I merged @jules32 minor edits from https://github.com/samanthacsik/samanthacsik.github.io/pull/27 and  made the following updates based on feedback:

> - Could you give a heads up about/introduce the wiki early in intro, saying something like "I documented what I learned and did throughly on the wiki, for my own/open science benefit and so that the Openscapes team can maintain". This could also be in a "TL;DR" type thing at the top. (I love that you got further into the wiki at the end!)

Yes! I added a TL;DR at the end of the intro paragraph that reads: 

***TL;DR** I rebuilt the [Openscapes website](https://openscapes.org/) using [Quarto](https://quarto.org/) and documented a lot of what I learned and how I did it in the website’s [GitHub Wiki](https://github.com/Openscapes/website-new/wiki). This was not only a super helpful practice for me in my own open science learning journey, but also for the Openscapes team as they take over continued maintenance of the site. I reflect on some of the most memorable moments of my first freelancing adventure, including creating responsive web pages, learning regex, and how open communication, collaboration, and cheering each other on can make any daunting task more approachable.*

> - love the greatest victory video! I also made me think a screenshot or video of the quarto site just after the blogdown screenshot for comparison?

I included a screenshot of the new landing page under the [General porting process](https://samanthacsik.github.io/posts/2023-07-31-openscapes-website/#general-porting-process-easy-stuff-first-then-lots-and-lots-of-learning-and-iterating) section -- this felt like a more natural place to include it than just after the blogdown screenshot, but I could be convinced to move it :)

@stefaniebutland you can find the link to the [post here](https://samanthacsik.github.io/posts/2023-07-31-openscapes-website/) (still set to draft so it's won't show up on my listing) and the [`index.qmd` here](https://github.com/samanthacsik/samanthacsik.github.io/blob/main/posts/2023-07-31-openscapes-website/index.qmd). The publish date (7/31/23) was just a placeholder -- I can update this in both the post directory name and the date listed in the yaml whenever we're ready to publish. 

Thanks so much to the both of you for taking the time to look things over!!

**Comment by stefaniebutland** [comment]
@samanthacsik this is such a beautiful post and I have zero suggestions! Let it out into the world and tag me here so I can publish our cross-post

I love this:

> we were dancing at the edge

**Comment by samanthacsik** [comment]
Thank you so much @stefaniebutland 🥲 !!! She's now live at https://samanthacsik.github.io/posts/2023-08-01-openscapes-website/!! and the post directory can be found at [posts/2023-08-01-openscapes-website](https://github.com/samanthacsik/samanthacsik.github.io/tree/main/posts/2023-08-01-openscapes-website). Let me know if you need my help with anything!

**Comment by stefaniebutland** [comment]
Woohoo! Our [cross-post pull request](https://github.com/Openscapes/website-new/pull/50) ([preview](https://deploy-preview-50--dreamy-solstice-e71879.netlify.app/blog/2023-08-03-openscapes-website/)) is ready to publish and toot Thurs Aug 3 (we just published a post today so will wait).

TIL from your post @samanthacsik that we can add ORCID to authors and have the link appear, without me changing anything in your post YAML, and citation appears too.

**Comment by stefaniebutland** [comment]
Published! https://openscapes.org/blog/2023-08-03-openscapes-website/
